src/BenchmarkDotNet.Annotations/Attributes/BenchmarkNameAttribute.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace BenchmarkDotNet.Attributes
+{
+    [AttributeUsage(AttributeTargets.Class)]
+    public class BenchmarkNameAttribute : Attribute
```

Comment:
I think `DescriptionAttribute` would be a better name. It matches the `Description` property of `BenchmarkAttribute`.